package-lock.json

```diff
@@ -35479,6 +35497,15 @@
         "@opentelemetry/api": ">=1.0.0 <1.9.0"
       }
     },
+    "packages/opentelemetry-resources/node_modules/@opentelemetry/api": {
+      "version": "1.4.1",
```

Comment:
I've seen this old one kicking around in other PR's package-lock files as well.

Unrelated to this PR, but I think it is due to this challenging dep in the 'resources' package:

```
  "devDependencies": {
    "@opentelemetry/api": ">=1.0.0 <1.9.0",
    "@opentelemetry/resources_1.9.0": "npm:@opentelemetry/resources@1.9.0",
```